docs/index.md

```diff
@@ -1,715 +1,19 @@
-X-Powered-By: Express
-Fiware-Correlator: 3a3bacd0-5bd1-450c-99e1-0dc33efe6f5f
-ETag: W/"2-vyGp6PvFo4RvsFtPoIWeCReyIC8"
-```
-
-#### Get a device (Optional)
-
-Request
-
-```bash
-curl -s -X GET \
-  'https://iot-ready-ul.lab.fiware.org:24061/iot/devices/UL-MQTT-Device' \
-  -H 'FIWARE-Service: ul' \
-  -H 'FIWARE-ServicePath: /mqtt' | python -m json.tool
-```
-
-Response
-
-```json
-{
-    "attributes": [
-        {
-            "name": "count",
-            "object_id": "c",
-            "type": "Integer"
-        }
-    ],
-    "commands": [],
-    "device_id": "UL-MQTT-Device",
-    "entity_name": "IoT-Ready-Entity",
-    "entity_type": "IoT-Ready-Type",
-    "lazy": [],
-    "protocol": "PDI-IoTA-UltraLight",
-    "service": "ul",
-    "service_path": "/mqtt",
-    "static_attributes": [],
-    "transport": "MQTT"
-}
-```
-
-#### Send the measurement
-
-Request
-
-```bash
-docker run -it --rm efrecon/mqtt-client pub -h iot-ready-ul.lab.fiware.org -p 27898 -m "c|1" -t "/ulmqttkey/UL-MQTT-Device/attrs"
-```
-
-#### Read the measurement
-
-Request
-
-```bash
-curl -s -X GET \
-  'https://iot-ready-ul.lab.fiware.org:21026/v2/entities?type=IoT-Ready-Type' \
-  -H 'FIWARE-Service: ul' \
-  -H 'FIWARE-ServicePath: /mqtt' | python -m json.tool
-```
-
-Response
-
-```json
-[
-    {
-        "TimeInstant": {
-            "metadata": {},
-            "type": "ISO8601",
-            "value": "2019-04-11T16:01:24.00Z"
-        },
-        "count": {
-            "metadata": {
-                "TimeInstant": {
-                    "type": "ISO8601",
-                    "value": "2019-04-11T16:01:24.484Z"
-                }
-            },
-            "type": "Integer",
-            "value": "1"
-        },
-        "id": "IoT-Ready-Entity",
-        "type": "IoT-Ready-Type"
-    }
-]
-```
+# About the program
+
+The FIWARE ecosystem means a straightforward business opportunity for IoT products: It is simple, the **FIWARE IoT
+stack** handles well-known IoT protocol standards and provides an enhanced OpenStack-based cloud environment plus a
+powerful set of standard REST APIs to ease the development of smart applications.
+
+A growing number of companies, including large corporations, startups, SMEs and individual developers worldwide are
+willing to provide their solutions based on FIWARE open-source [components](https://github.com/fiware/catalogue).
+
+So far, developers needed to search compatible IoT products and find out how to make them work. The certification of
+**FIWARE-ready** IoT Devices provides the answer as it is designed to demonstrate the ability of the IoT solutions to
+seamless interoperate with FIWARE platforms. The database of products and developers that have successfully been
+featured and certified as **FIWARE-ready** IoT Devices will be part of the FIWARE Marketplace, a global one-stop shop
+that gives visibility to a wide range of Powered by FIWARE solutions/platforms, **FIWARE-ready** technologies as well as
```

Comment:
Powered by FIWARE platforms... mmm it is hw in which you can deploy and execute the different FIWARE GEs

Reply:
**Won't Fix** - this terminology reflects the existing divisions of products found on `https://marketplace.fiware.org/`